Sources/ArcGISToolkit/Components/Augmented Reality/CalibrationView.swift

```diff
@@ -0,0 +1,170 @@
+                                rotateHeading(joystickHeadingDelta)
+                            }
+                            headingTimer = timer
+                            // Add the timer to the main run loop.
+                            RunLoop.main.add(timer, forMode: .default)
+                        }
+                        
+                        Button {
+                            let heading = viewModel.cameraController.originCamera.heading + 1
+                            updateHeading(heading)
+                        } label: {
+                            Image(systemName: "plus")
+                        }
+                    }
+                    VStack {
+                        Text("Elevation: \(viewModel.calibrationElevation?.rounded(.towardZero) ?? viewModel.cameraController.originCamera.location.z?.rounded(.towardZero) ?? 0, format: .number)")
+                        HStack {
+                            Button {
+                                updateElevation(-1)
+                            } label: {
+                                Image(systemName: "minus")
+                            }
+                            Slider(value: $elevationSliderValue, in: -20...20) { editingChanged in
+                                if !editingChanged {
+                                    elevationTimer?.invalidate()
+                                    elevationTimer = nil
+                                    elevationSliderValue = 0.0
+                                }
+                            }
+                            .onChange(of: elevationSliderValue) { elevation in
+                                guard elevationTimer == nil else { return }
+                                // Create a timer which rotates the camera when fired.
+                                let timer = Timer(timeInterval: 0.1, repeats: true) { [self] (_) in
+                                    updateElevation(joystickElevationDelta)
+                                }
+                                elevationTimer = timer
+                                // Add the timer to the main run loop.
+                                RunLoop.main.add(timer, forMode: .default)
+                            }
+                            Button {
+                                updateElevation(1)
+                            } label: {
+                                Image(systemName: "plus")
+                            }
+                        }
+                    }
+                }
+                .onDisappear {
+                    guard !viewModel.isCalibrating else { return }
+                    withAnimation(.easeInOut) {
+                        setBasemapOpacity(0)
+                    }
+                }
+                .frame(idealWidth: UIScreen.main.bounds.width/3)
+                .fractionPresentationDetents(0.25)
+                .padding()
+            }
+            .esriBorder()
```

Comment:
Currently this button is at lower left with no margin. As Ryan said, a button that shows a material overlay would be a good alternative.
![IMG_3499](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/9660181/68353919-64d7-4c4a-a79b-feef7f6ca848)